src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -124,6 +124,8 @@ public String getCustomMessage() {
         return customMessage;
     }
 
+	public SlackNotifier(){};
```

Comment:
Apologies for being pedantic, but:
* indented by a tab instead of spaces, which doesn't match the rest of the file
* redundant semi-colon on end of line
* personally, it's easier for me to see what's going on if the closing brace is on the next line (this is the least concerning, and I don't really care)

It would be nice to get this fixed and merged so we can have 1 less PR out there.